stdlib/universe/universe.flux

```diff
@@ -76,6 +76,7 @@ builtin tripleExponentialDerivative : (<-tables: [{B with _value: A}], n: int) =
 builtin union : (tables: [[A]]) => [A] where A: Record
 builtin unique : (<-tables: [A], ?column: string) => [A] where A: Record
 builtin window : (<-tables: [A], ?every: duration, ?period: duration, ?offset: duration, ?timeColumn: string, ?startColumn: string, ?stopColumn: string, ?createEmpty: bool) => [B] where A: Record, B: Record
+builtin windowInterval : (<-tables: [A], ?every: duration, ?period: duration, ?offset: duration, ?timeColumn: string, ?startColumn: string, ?stopColumn: string, ?createEmpty: bool) => [B] where A: Record, B: Record
```

Comment:
I thought that we were introducing this through a planner rule. Is this just for testing or is this a new function?

Reply:
Yes, I needed to have a separate procedure spec for the planner to be aware of the new transformation. It required so much duplication that I decided to temporarily add a new transformation (windowInterval). We can delete the old window transformation and promote this one once the feature flag for interval is rolled out.

Reply:
my WIP for the new feature flag https://github.com/influxdata/idpe/pull/9730

Reply:
If the goal is to replace it with a new procedure spec, this isn't required. The function name only needs to be specified if you plan on invoking it from directly within a flux script like `|> windowInterval`. If the plan is just to replace the window transformation with a new procedure spec, then you can omit this.

Reply:
I've removed the unnecesary pieces from the windowInterval transformation so that its not user facing any longer.